app/src/main/java/com/kickstarter/ui/activities/BackingActivity.java

```diff
@@ -83,6 +84,7 @@ public void onCreate(final @Nullable Bundle savedInstanceState) {
     this.rewardsItemRecyclerView.setAdapter(rewardsItemAdapter);
     final LinearLayoutManager layoutManager = new LinearLayoutManager(this);
     this.rewardsItemRecyclerView.setLayoutManager(layoutManager);
+    this.rewardsItemRecyclerView.addItemDecoration(new RewardItemDecorator(getDrawable(R.drawable.divider_grey_500_horizontal)));
```

Comment:
the color of the divider is `ksr_grey_300` according to Abstract. also, instead of passing in this drawable, we could just draw a 1dp  line

Reply:
Just for curiosity, what would be advantage of drawing the line instead of passing in the drawable? We are using the same divider in other places, wouldn't be better to have one source of truth? 🤔

Reply:
but the truth is in the abstract 👀 the truth will set you free

Reply:
also i suggest just drawing a line of that color or else you should make a new divider drawable of that color 😛

Reply:
Oh, by source of truth I meant in the code. Like if we need to change the color of the dividers in the future, we would need to change it in one place - the drawable, instead of going to individual files and changing the color of the drawn lines. Does it make sense?
And yeah, I've created a `divider_grey_300_horizontal` 🙈